var/spack/repos/builtin/packages/ldak/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Ldak(Package):
+    """LDAK is a software package for analyzing GWAS data"""
+
+    homepage = "http://dougspeed.com/ldak/"
+    url      = "http://dougspeed.com/wp-content/uploads/source.zip"
```

Comment:
It's a shame there isn't a URL that contains the version number, but I couldn't find one either.